src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../index.css';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="row1">
+      <Button name="AC" color="lightgray" />
+      <Button name="+/-" color="lightgray" />
+      <Button name="%" color="lightgray" />
+      <Button name="/" />
+    </div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them.